ubuntu/dlang.docker

```diff
@@ -23,9 +23,19 @@ RUN set -ex && \
 	&& update-alternatives --install "/usr/bin/ld" "ld" "/usr/bin/ld.gold" 20 \
 	&& update-alternatives --install "/usr/bin/ld" "ld" "/usr/bin/ld.bfd" 10
 
+RUN groupadd --gid 1000 dlang \
+  && useradd --uid 1000 --gid dlang --shell /bin/bash --create-home dlang
+
+RUN mkdir ${DPATH}\
+    && chown dlang ${DPATH}
+
+USER dlang
+
 RUN set -ex && \
 	curl -fsS https://dlang.org/install.sh | bash -s ${D_VERSION} -p ${DPATH}
 
+USER root
```

Comment:
Don't you want to run code with the user `dlang` then?

Reply:
By setting user with id 1000 (dlang) as default user for the image, this will immediately break usages of this image (like the example-app from this repository).
On the other hand, Jenkins will start any container (npm/maven/dub) with default user id 1000.

Therefore this pr enables a new use case without breaking old use cases. User id 1000 is now there and fully working but root is still stays and is the default.